scripts/classes.js

```diff
@@ -0,0 +1,70 @@
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+
+class BookList {
+  constructor() {
+    this.title = document.getElementById('title');
+    this.author = document.getElementById('author');
+    this.addBtn = document.getElementById('add-btn');
+    this.books = document.getElementById('books');
+    this.booksArr = [];
+    this.loadBooks();
+    this.addBtn.addEventListener('click', () => this.addItem());
+    document.addEventListener('DOMContentLoaded', () => this.loadBooks());
+  }
+
+  removeItemAt(index) {
+    this.booksArr.splice(index, 1);
+  }
+
+  showBooks() {
+    this.books.innerHTML = '';
+    this.booksArr.forEach((book, index) => {
+      const appendBook = `
+            <p><strong>"${book.title}"</strong> by ${book.author}</p>
+            <button id="btn-${index}">remove</button>
+        `;
+      const bookContainer = document.createElement('div');
+
+      bookContainer.id = `book-${index}`;
+      bookContainer.classList.add(`${(!index % 2) ? 'g-bg' : 'w-bg'}`);
+      bookContainer.innerHTML = appendBook;
+      this.books.appendChild(bookContainer);
+
+      const removeBtn = document.getElementById(`btn-${index}`);
+      removeBtn.addEventListener('click', () => {
+        this.removeItemAt(index);
+        localStorage.setItem('booksData', JSON.stringify(this.booksArr));
+        this.showBooks();
+      });
+    });
+  }
+
+  loadBooks() {
+    if (localStorage.booksData) {
+      this.booksArr = JSON.parse(localStorage.booksData);
+      this.showBooks();
+    } else {
+      this.booksArr = [];
+    }
+  }
+
+  addItem() {
+    if (this.title.value !== '' && this.author.value !== '') {
+      const theBook = {
+        title: this.title.value,
+        author: this.author.value,
+      };
+
+      this.booksArr.push(theBook);
+      this.title.value = '';
+      this.author.value = '';
+      localStorage.setItem('booksData', JSON.stringify(this.booksArr));
+      this.showBooks();
+    }
```

Comment:
- [ ] As mentioned by the privous code reviewer I think it would be great if you fix the problem because I still can't add a new book so kindly fix the issue to let your project works as expected ✅🙏🏻 I think I saw another file called script.js and I am not sure what is that if you refactor your code so I think you don't need the another file 🙏🏻💯💯



https://user-images.githubusercontent.com/58553711/226908022-3363072f-717e-456e-86e3-7612c837b462.mp4